_episodes/04-pipefilter.md

```diff
@@ -256,10 +256,14 @@ the output of `head` must be the file with the fewest lines.
 > incorrect results and/or delete
 > the contents of `lengths.txt`.
 {: .callout}
-
 > ## What Does `>>` Mean?
 > We have seen the use of `>`, but there is a similar operator `>>` which works slightly differently.
-> By using the `echo` command to print strings, test the commands below to reveal the difference
+> First, let's try using the `echo` command to print strings, e.g.: 
+> ~~~
+> $ The echo command prints text
+> ~~~
```

Comment:
This is a hybrid of the output and the command.
The command should be
```
$ echo The echo command prints text
```

and the corresponding output should be 
```
The echo command prints text
```

Reply:
Yep, what a daft mistake. Well spotted.

Reply:
Great, thanks. I think it would also be good to show the expected output in a code block.